reflex/__init__.py

```diff
@@ -82,26 +85,30 @@
     "slider",
     "spacer",
     # "strong" (bold?)
+    "stack",
     "switch",
     "table",
     "tabs",
     "text",
     "text_area",
-    # "text_field" (input)
+    "text_field",
```

Comment:
This should be `input`, we shouldn't have `rx.text_field`

Reply:
it is done